app/helpers/nationalities_helper.rb

```diff
@@ -1,15 +1,24 @@
 # frozen_string_literal: true
 
 module NationalitiesHelper
-  def format_default_nationalities(nationalities)
+  def format_nationalities(nationalities, description: true, empty_option: false)
     formatted_nationalities = []
```

Comment:
I'm wondering whether it's worth saving on some computation by returning from this method early if the nationalities have already been formatted for the view. Although could be premature to do it now since the total nationalities is still small but maybe something to think about if it grows.

Reply:
I think I'm going to leave this for now as this method is called from the same view with different sets of nationalities, so memoization would need a bit of a re-work. Let's definitely keep in mind for if the list / number of calls expands 👍